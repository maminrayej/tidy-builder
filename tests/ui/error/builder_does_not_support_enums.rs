#[derive(tidy_builder::Builder)]
enum MyEnum {
    A,
    B,
}

fn main() {}