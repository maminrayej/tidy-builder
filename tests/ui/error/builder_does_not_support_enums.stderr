error: Builder does not support enums
 --> tests/ui/error/builder_does_not_support_enums.rs:2:1
  |
2 | enum MyEnum {
  | ^^^^
